managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -1020,6 +1027,16 @@ export default class ClusterFields extends Component {
     }
   }
 
+  /**
+   * This method is used to disable the ClientToNodeTLS field initially.
+   * 
+   * @param isFieldReadOnly If true then readonly access.
+   * @param enableNodeToNodeEncrypt NodeToNodeTLS state.
+   */
+  clientToNodeEncryptField(isFieldReadOnly, enableNodeToNodeEncrypt) {
+    return isFieldReadOnly ? isFieldReadOnly : !enableNodeToNodeEncrypt;
```

Comment:
Can change to `return isFieldReadOnly || !enableNodeToNodeEncrypt;`

Reply:
Done.